CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -252,7 +255,7 @@ public function buildQuickForm() {
       $this->add('select', 'payment_instrument_id',
         ts('Payment Method'),
         array('' => ts('- select -')) + CRM_Contribute_PseudoConstant::paymentInstrument(),
-        FALSE,
+        TRUE,
```

Comment:
@pradpnayak I have set required = TRUE because this field is present only in backoffice form not on live mode, see the check is made on line 252. Also tested in my local and it's working fine w/o live mode.